### DISCUSSION TITLE
Issues connecting Ambient Occluson map in the Model Editor

### DISCUSSION THREAD

**Discussion by Pedergnana** [google/model-viewer#2665] [Q&A]
The ambient occlusion if I use a RGP image where R channel is the AO, the model editor is not working properly anymore like before the last update. The entire model turns black. Any suggestions?

**Comment by elalish** [comment]
I said detailed repro steps! Link your model here and any textures, then list each thing you click in order and put screenshots to describe where the problem is. Is the model black when first loaded or only after changing something?

**Reply by Pedergnana** [reply]
Here all the step:
- starting from an empty GLB file;
- import the GLB in Model Editor (image1);
- connecting the color map, is working (image2);
- connect roughness/metal map, is working (image3);
- connecting normal map, is working (image4);
- connecting AO, is NOT working. Everything turns darker (image5);
- image 6 to show the map. The AO map is in the red channel. 2 weeks ago this model was working perfectly. 

Thank you for your support @elalish!   

![1](https://user-images.githubusercontent.com/88549776/128545365-592226b6-09b2-48a2-9492-457acd80e48e.JPG)
![2](https://user-images.githubusercontent.com/88549776/128545369-b0db60e1-33e5-4140-9658-df7715e50ca4.JPG)
![3](https://user-images.githubusercontent.com/88549776/128545370-37c4a889-5a3f-442c-bfa2-0f581395bd8d.JPG)
![4](https://user-images.githubusercontent.com/88549776/128545374-60fc991b-0f75-4853-b42c-6fdf9ea71e91.JPG)
![5](https://user-images.githubusercontent.com/88549776/128545377-323f14d3-a4ae-4749-82e2-858bc31fcc5a.JPG)

HERE, THE TEXTURES: 
![lambert1_baseColor](https://user-images.githubusercontent.com/88549776/128545462-d791f6c8-c126-4125-8c9e-818cccd3cdee.png)
![lambert1_normal](https://user-images.githubusercontent.com/88549776/128545468-17a8b5b6-3c09-4668-9230-30989b19f32d.png)
![lambert1_occlusionRoughnessMetallic](https://user-images.githubusercontent.com/88549776/128545470-18c197eb-81a7-4bf0-9f8d-3ed64e3032b6.png)

**Reply by elalish** [reply]
Thank you! I can't seem to repro with any of my own models. Can you put a link your chair model here?

**Reply by Pedergnana** [reply]
Here the .zip of this test model. 
[Seat_test.zip](https://github.com/google/model-viewer/files/6947362/Seat_test.zip)


Thank you!
Lorenzo

**Reply by Pedergnana** [reply]
If I try to export the .glb file from ModelEditor and import it back, I have this error: 

![Capture](https://user-images.githubusercontent.com/88549776/129090026-c0cbe933-a0cb-4edf-8f86-c03017eb6c2a.JPG)

I'm using a Lambert shader. Any suggestions? @elalish

**Reply by elalish** [reply]
sounds like it might be a bug in three's GLTFExporter? Also, I don't think a Lambert shader will have much hope of making a good glTF; try a meshPhysicalMaterial or meshStandardMaterial.
